docs/components/modeler/forms/form-element-library/forms-element-library-textfield.md

```diff
@@ -28,6 +28,14 @@ A text field allowing the user to read and edit textual data.
   - **Prefix**: Adds an appendage before the input. Can either be an [expression](../../feel/language-guide/feel-expressions-introduction.md), plain text, or [templating syntax](../configuration/forms-config-templating-syntax.md).
   - **Suffix**: Adds an appendage after the input. Can either be an [expression](../../feel/language-guide/feel-expressions-introduction.md), plain text, or [templating syntax](../configuration/forms-config-templating-syntax.md).
 
+:::info
+
+The phone pattern adheres to the international [E.164](https://en.wikipedia.org/wiki/E.164) standard, omitting spaces. Example: `+491234567890`
```

Comment:
I'm always wary of linking to Wikipedia, so this may be a better link here https://www.itu.int/rec/T-REC-E.164/

Reply:
@christinaausley What do you think about using [Twilio's docs](https://www.twilio.com/docs/glossary/what-e164)?
It looks like it's way better explained than both our links

Reply:
Excellent! Adjusting in my latest commit, one second.